src/views/Admin.js

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback } from 'react';
+import { Just } from 'folktale/maybe';
+import { Grid } from 'indigo-react';
+import * as need from 'lib/need';
+
+import View from 'components/View';
+import { ForwardButton } from 'components/Buttons';
+
+import { ROUTE_NAMES } from 'lib/routeNames';
+import { eqAddr } from 'lib/wallet';
+
+import { useHistory } from 'store/history';
+import { useWallet } from 'store/wallet';
+import { usePointCursor } from 'store/pointCursor';
+import { usePointCache } from 'store/pointCache';
+import FooterButton from 'components/FooterButton';
+
+export default function Admin() {
+  const history = useHistory();
+  const { urbitWallet, wallet } = useWallet();
+  const { pointCursor } = usePointCursor();
+  const { pointCache } = usePointCache();
+
+  const point = need.point(pointCursor);
+  const pointDetails = need.fromPointCache(pointCache, point);
+  const userAddress = need.wallet(wallet).address;
+
+  const goRedownload = useCallback(() => history.push(ROUTE_NAMES.REDOWNLOAD), [
+    history,
+  ]);
+
+  const goReticket = useCallback(() => history.push(ROUTE_NAMES.RETICKET), [
+    history,
+  ]);
+
+  const goEditPerms = useCallback(() => history.push(ROUTE_NAMES.PERMISSIONS), [
+    history,
+  ]);
+
+  const goTransfer = useCallback(() => history.push(ROUTE_NAMES.TRANSFER), [
+    history,
+  ]);
+
+  const canDownloadPassport = Just.hasInstance(urbitWallet);
+  const isOwner = eqAddr(userAddress, pointDetails.owner);
+
+  return (
+    <View>
+      <Grid className="pt3">
+        <Grid.Item full>
```

Comment:
For these, I would recommend a syntax like

```jsx
<Grid.Item full as={ForwardButton} disabled={!canDownloadPassport} onClick={...} {...theRestOfTheButtonProps} />
```

this way the button itself becomes the grid item (using the grid layout classes), which makes it automatically stretch the width of the grid and respect the other grid-specific layouts. buttons _do_ automatically stretch, but other elements might not, so I've standardized on making sure that all of the actual elements we care about behave as grid or flex items, which removes any uncertainty around how they'll be laid out.